### PR TITLE
Upgrade to Covalent 1.0.0-beta.3-2, Angular 4.0.0, Electron 1.6.6

### PR DESCRIPTION
**Pull request by jeremysmartt**
## Description

This PR upgrades Covalent Electron to use dependencies Covalent 1.0.0-beta.3-2, Angular 4.0.0, Electron 1.6.6 and the sub dependencies of those.

### What's included?

- 

#### Test Steps

- [ ] Checkout branch
- [ ] run `npm run test`
- [ ] See tests pass
- [ ] delete dist directory
- [ ] run `npm run live-reload -- --openDevTools`
- [ ] make changes and see them live reload
- [ ] stop app
- [ ] run `npm run package`
- [ ] execute dist-app/Covalent-darwin-x64/Covalent.app
- [ ] see app runs

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -9,7 +9,6 @@ import { CovalentCoreModule } from '@covalent/core';
 import { CovalentHttpModule } from '@covalent/http';
```
Comment (thomaslennan) [inline src/app/app.module.ts]: I saw an error in the console regarding the need to include either BrowserAnimationsModule or NoopAminationsModule. Wondering if you want to include that in the app.modules.

Comment (emoralesb05) [inline src/app/app.module.ts]: You are correct sir. It should include `BrowserAnimationsModule`

Comment (jeremysmartt) [inline src/app/app.module.ts]: nice catch @thomaslennan

Comment (jeremysmartt) [inline src/app/app.module.ts]: @thomaslennan Fix checked in

**Pull request merged**